src/github.com/getlantern/lantern-mobile/app/build.gradle

```diff
@@ -118,4 +119,14 @@ dependencies {
     compile('com.crashlytics.sdk.android:crashlytics:2.5.3@aar') {
         transitive = true
     }
+
+    apt "org.androidannotations:androidannotations:" + androidAnnotationsVersion
+    compile("org.androidannotations:androidannotations-api:" + androidAnnotationsAPIVersion )
+}
+
+apt {
+    arguments {
+        androidManifestFile variant.outputs[0].processResources.manifestFile
+        resourcePackageName 'org.getlantern.lantern'
```

Comment:
It looks like there might be an issue with the apt plugin (necessary to make annotations work with Android Studio) settings. What error is Android Studio giving you? Should be more info here: https://bitbucket.org/hvisser/android-apt